Source/WebKit/Platform/cocoa/AssertionCapability.h

```diff
@@ -43,15 +43,15 @@ class AssertionCapability final : public ExtensionCapability {
 
     // ExtensionCapability
     String environmentIdentifier() const final { return m_environmentIdentifier; }
-    RetainPtr<_SECapability> platformCapability() const final;
+    RetainPtr<BEProcessCapability> platformCapability() const final;
 
 private:
     String m_environmentIdentifier;
     String m_domain;
     String m_name;
     BlockPtr<void()> m_willInvalidateBlock;
     BlockPtr<void()> m_didInvalidateBlock;
-    RetainPtr<_SECapability> m_capability;
+    RetainPtr<BEProcessCapability> m_capability;
```

Comment:
Since both `AssertionCapability` and `MediaCapability` now have a stored `m_capability`, we could just move this to the base class.